pyproject.toml

```diff
@@ -7,6 +7,7 @@ readme = "README.md"
 
 [tool.poetry.dependencies]
 python = "^3.9"
+deltalake = "^0.7.0"
```

Comment:
Don't think we want to add deltalake as a dependency for this particular project.

Reply:
@MrPowers Yeah sorry, I will remove it. I was just putting this in WIP so I could test the `ruff` git workflow.

Reply:
@danielbeach - yea, keep iterating.  I normally have to tweak these workflows like 10 times before I get them working, haha.  Let me know if you need any help.